pyproject.toml

```diff
@@ -64,10 +64,11 @@ optional-dependencies.dev = [
     "xmlschema",
 ]
 urls.Changelog = "https://docs.pytest.org/en/stable/changelog.html"
+urls.Contact = "https://docs.pytest.org/en/stable/contact.html"
+urls.Funding = "https://docs.pytest.org/en/stable/sponsor.html"
 urls.Homepage = "https://docs.pytest.org/en/latest/"
 urls.Source = "https://github.com/pytest-dev/pytest"
 urls.Tracker = "https://github.com/pytest-dev/pytest/issues"
-urls.Twitter = "https://twitter.com/pytestdotorg"
```

Comment:
Do we not want bsky/Mastodon here?

Reply:
I added the "Contact" link instead which lists those (and also IRC, Discord, ...) instead of adding them all to PyPI individually.